index.html

```diff
@@ -14,6 +16,77 @@
 </head>
 
 <body>
-<!-- Use your own code or past solution for Great Idea Web Page HTML here! -->
+  <div class="main">
+    <header>
+      <nav>
+        <a href="services.html">Services</a>
+        <a href="">Product</a>
+        <a href="">Vision</a>
+        <a href="">Features</a>
+        <a href="">About</a>
+        <a href="">Contact</a>
+  
+        <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+
+      </nav>
+      <div class="headMain">
+        <div class="headTitle">
+          <h1>
+            Innovation<br> On
+            <br> Demand
+          </h1>
+          <div class="button">
+            Get Started
+          </div>
+        </div>
+        <img class='headerImg' src="img/header-img.png" alt="Image of a code snippet.">
+      </div>
+    </header>
+  
+    <content>
+      <div class="content">
```

Comment:
You may want to rename this class as something over than `content` since you are using a content tag above - mainly for organization and clarification.